test/integration/app/wallets/create-on-chain-address.spec.ts

```diff
@@ -8,6 +8,31 @@ import { AccountsRepository } from "@services/mongoose"
 import { createRandomUserAndBtcWallet } from "test/helpers"
 
 describe("onChainAddress", () => {
+  it("can apply requestId as idempotency key when creating new address", async () => {
```

Comment:
should not we use `getLastOnChainAddress` too?

Reply:
This one was just to explicitly test the requestId idempotency behavious that we haven't exposed at the graphql level. This would've otherwise been an e2e test. The rest of happy path create/getLast tests are in bats tests.

Reply:
This was just to specifically test the idempotency behaviour of that `requestId` param that we haven't exposed at the graphql layer. The rest of happy path tests for create/getLast should be in e2e bats tests